### ISSUE TITLE
Expose static version to the internet

### ISSUE THREAD

**Issue by sefsh** [jparyani/mediawiki-sandstorm#8]
It would be _really_ useful if the Sandstorm version of MediaWiki was able to render a static version of the wiki for outside reach, much like what the Sandstorm versions of Wordpress, Ghost or Davros does. Shareable link creation is of course possible, but those can't be bound to a domain or subdomain.

**Comment by jparyani** [comment]
Ya I agree, this would be awesome. It would add a lot of security and performance benefits, with little downside. I'm not sure when I'll have time to work on it though.

**Comment by sefsh** [comment]
Just making sure, it's not possible right now to get a bindable URL to the app as of now right?

**Comment by jparyani** [comment]
Unfortunately, no. Your best bet for now is to setup a redirect, but that has the obvious downside of confusing users when they get redirected to this weird URL.

**Comment by sefsh** [comment]
I see! For me the wiki need to be exposable to be useful. Sadly, the state of static output methods seems to be a horror story. About [DumpHTML](https://www.mediawiki.org/wiki/Extension:DumpHTML):

> The only alive human known to have managed using dumpHTML with success is Kelson

:flushed:

**Comment by razlupercio** [comment]
Isn't possible to publish static content using a publicId as stated [here](https://docs.sandstorm.io/en/latest/developing/web-publishing/)?

**Comment by jparyani** [comment]
Yes, it is definitely possible. It's just that it would take a fair amount of work to export the site as static content and wire it all up so that the admins/users can control/see it. Also, we'd have to answer the question of how users would be given the option to login (the answer may be that they aren't, but that's a bit sad).
